libeos-updater-util/flatpak.c

```diff
@@ -921,10 +921,10 @@ read_flatpak_ref_actions_from_node (JsonNode      *node,
                                     EOS_UPDATER_ERROR,
                                     EOS_UPDATER_ERROR_UNKNOWN_ENTRY_IN_AUTOINSTALL_SPEC))
             {
-              g_debug ("%s while parsing %s. Skipping this action and it "
-                       "will not be reapplied later. System may be in an "
-                       "inconsistent state from this point forward.",
-                       local_error->message, filename);
+              g_message ("%s while parsing %s. Skipping this action and it "
```

Comment:
I’d go with `g_warning()`. Was there any discussion about the logging level in use for these messages in the initial PR?

Reply:
I'm not sure. I think `debug` was suggested somewhere and then it got copied.